ButtonStateless/index.jsx

```diff
@@ -31,6 +31,7 @@ const Button = ({
   focused,
   hovered,
   fillContainer,
+  icon,
```

Comment:
I don't think the `Button` should need to know there is an `Icon` in it. The `Icon` should have no background by default.